src/main/java/com/project/momo/entity/Gathering.java

```diff
@@ -0,0 +1,36 @@
+package com.project.momo.entity;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "GATHERING")
+public class Gathering extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(length = 45)
+    private String name;
+
+    @ManyToOne
+    @JoinColumn(name = "code_key")
+    private Code code;
+
+    @Column(name = "gather_at", columnDefinition = "TIMESTAMP")
+    private LocalDateTime gatherAt;
+
+    @Column(length = 45)
+    private String location;
+
+    @Column(name = "max_number", columnDefinition = "SMALLINT")
+    private int max_number;
```

Comment:
```suggestion
    private int maxNumber;
```
자바 네이밍의 기본 컨벤션은 카멜 케이스입니다 :)